projects/sd-jwt-rs.md

```diff
@@ -0,0 +1,82 @@
+# Project Name
+
+SD-JWT Rust Reference Implementation
```

Comment:
Let's start with a reference implementation, but the long-term goal for DSR is to have a real-life, production-ready, high-performance implementation in Rust.